packages/utils/odsp-doclib-utils/src/test/odspErrorUtils.spec.ts

```diff
@@ -137,6 +137,20 @@ describe("OdspErrorUtils", () => {
             assert(isILoggingError(error));
             assert.equal(error.getTelemetryProperties().innerMostErrorCode, "bar");
         });
+        it("error response with redirect location", () => {
+            const responseText = '{ "error": { "message":"hello", "@error.redirectLocation":"url" } }';
+            const error = createOdspNetworkError(
+                "Test Message",
+                404,
+                undefined, /* retryAfterSeconds */
+                undefined, /* response */
+                responseText,
+            );
+            assert(error.errorType === DriverErrorType.fileNotFoundOrAccessDeniedError,
+                 "Error should be a fileNotFoundOrAccessDeniedError");
+            assert(error.redirectLocation === "url", "redirect location is wrong");
+            assert(isILoggingError(error));
+        });
```

Comment:
And lastly do `assert(error.getTelemetryProperties().redirectLocation === undefined)`